docs/ref/contrib/gis/db-api.txt

```diff
@@ -300,15 +298,15 @@ The following table provides a summary of what GIS-specific aggregate functions
 are available on each spatial backend. Please note that MySQL does not
 support any of these aggregates, and is thus excluded from the table.
 
-====================================  =======  ======  ==========
-Aggregate                             PostGIS  Oracle  SpatiaLite
-====================================  =======  ======  ==========
-:class:`Collect`                      X                (from v3.0)
-:class:`Extent`                       X        X       (from v3.0)
-:class:`Extent3D`                     X
-:class:`MakeLine`                     X
-:class:`Union`                        X        X       X
-====================================  =======  ======  ==========
+===============================================  =======  ======  ==========
+Aggregate                                        PostGIS  Oracle  SpatiaLite
+===============================================  =======  ======  ==========
+:class:`~django.contrib.gis.db.models.Collect`   X                (from v3.0)
```

Comment:
feel free to use `.. currentmodule:: django.contrib.gis.db.models ` above the table to avoid having to modify the references in it